src/pipelines/epidemiology/ro_covid19romania.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import Any, Dict, List
+from pandas import DataFrame, concat, merge
+from lib.io import read_file
+from lib.pipeline import DataSource
+from lib.time import datetime_isoformat
+from lib.utils import table_rename, table_multimerge
+
+
+class Covid19RomaniaDataSource(DataSource):
+    def parse(self, sources: List[str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
+
+        data_list = []
+        for idx, var in enumerate(["total_recovered", "current_intensive_care"]):
+            df = DataFrame.from_records(json.load(open(sources[idx]))["values"])
```

Comment:
It appears we're just iterating over the first two values in `sources`. Any reason we can't just do a `for var in sources[:2]` here?

Reply:
If we add another var, then that's one more line to potentially forget to update.